lms/static/js/edxnotes/plugins/llm_summarize.js

```diff
@@ -68,17 +68,113 @@
                         background-color: white;
                         border: 1px solid gray;
                     }
+
+                    .summarize-loader-wrapper {
+                        display: none;
+                    }
+
+                    .summarize-loader-wrapper.show {
+                        display: block
+                    }
+                    // Defining content loader since fontawesome icons don't work
+                    // inside the annotatorjs modal.
+                    .loader {
+                        width: 5em !important;
+                        border-color: red !important;
+                    }
+                    @keyframes rotation {
+                    0% {
+                        transform: rotate(0deg);
+                    }
+                    100% {
+                        transform: rotate(360deg);
+                    }
                 `;
                 document.head.appendChild(style);
-                var annotator = this.annotator;
+                this.modifyDom(this.annotator);
+                const summarizeButton = document.getElementById('summarizeButton');
+                summarizeButton.addEventListener('click', function(ev) {
+                    document.querySelector('.annotator-outer.annotator-editor').setAttribute('is_summarizing', true);
```

Comment:
Please, use the same element selector function thought the file, so in case that something in document API changes, we will apply the same change to everything and all will behave in the same way. @alexjmpb